masterlist.yaml

```diff
@@ -2219,6 +2219,10 @@ plugins:
       - link: 'https://www.nexusmods.com/oblivion/mods/3979/'
         name: 'Harvest Containers'
     msg: [ *alreadyInOOO ]
+  - name: 'Harvest\[Containers\]\.es[mp]'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[Harvest Containers xOBSE](https://www.nexusmods.com/oblivion/mods/52317/)' ]
```

Comment:
Use instead would be a better here, and you could add a condition to check for OBSE.

Reply:
Harvest Containers already requires OBSEv15 so I'll check for xOBSE 22.6 or greater.